File: day 36/cw/cw.py
def manual_capitalize(user_str):
    return user_str[0].upper() + user_str[1:].lower()

print(manual_capitalize("hello world"))
print(manual_capitalize("pYTHON"))
print(manual_capitalize("geORGia"))
print(manual_capitalize("123abc"))
print(manual_capitalize("hELLO"))


def lower_or_upper(user_str, choice):
    if choice == "upper":
        print(user_str.upper())
    elif choice == "lower":
        print(user_str.lower())
    else:
        print("wrong choice")

lower_or_upper("Hello, World!", "upper")
lower_or_upper("Python Programming", "lower")
lower_or_upper("Test String", "random")


# es ver gavige

def big_sentance(name, surname, age, academy, role):
    f"Hello, my name is {name}, my surname is {surname}. I am {age} years old. I study at {academy}, my role is {role}."


print(big_sentance("dato","Zhamerashvili",11,"goal-oriented acdemy","student"))

File: day 73/cw/cw.js
let emptyString = ""

do{
    emptyString += "m"
}while(emptyString.length < 20)

console.log(emptyString)

let student = {
    name: "David",
    age: 12,
    grade: "7th"
    };

    for (let key in student) {
    console.log(key + ": " + student[key]);
    }

    for (let i = 1; i <= 100; i++) {
            if (i % 5 === 0 && i % 7 === 0) {
        continue;
    }
    console.log(i);
}
